spring-cloud-gcp-pubsub/pom.xml

```diff
@@ -42,5 +42,34 @@
 			<artifactId>spring-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!-- SI channel adapters -->
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-core</artifactId>
```

Comment:
I don't mind to keep everything in this `spring-cloud-gcp` project. Only my concern that it might be better to keep SI Channel Adapters as a separate module, therefore separate artifact (jar) in the end. 
The name I suggest is `spring-cloud-gcp-integrtaion` because we might have adapters not only just for PubSub.
Thanks